### ISSUE TITLE
Request: Have listings in 'overview' pages.

### ISSUE THREAD

**Issue by bobdenotter** [bolt/docs#506]
It'd be very helpful to users if pages that have 'children' would show those pages in the right hand side. Not just in the navigation tree. 

For example on pages like this: 

https://docs.bolt.cm/3.1/internals

It should look like this: 

![screen_shot_2016-09-23_at_16_33_57](https://cloud.githubusercontent.com/assets/1833361/18789607/dc439c38-81ab-11e6-813f-de17eedac0d7.png)

**Comment by CarsonF** [comment]
With #535 
```twig
{% for subPage in page.subPages %}
  <a href="{{ subPage.slug }}">{{ subPage.title }}</a>
{% endfor %}
```

**Comment by bobdenotter** [comment]
Fixed by Carson! (mostly in #535)
